python_practice/P003.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""make a janken program (vs a 'computer' <- randomize the computer's attempts),
+ whoever wins three times is the winner"""
+from random import randint
+
+
+def janken(computer, player):
+    if computer == "gu":
+        if player == "gu":
+            return "Even"
+        if player == "choki":
+            return "You lose"
+        if player == "pa":
+            return "You win"
+    if computer == "choki":
```

Comment:
you can make this an elif so that when `computer == "gu"` this condition won't be checked anymore